solidity/ecdsa/test/WalletRegistry.Wallets.test.ts

```diff
@@ -47,8 +227,54 @@ describe("WalletRegistry - Wallets", async () => {
       })
 
       it("should return true", async () => {
-        await expect(await walletRegistry.isWalletRegistered(publicKeyHash)).to
-          .be.true
+        await expect(await walletRegistry.isWalletRegistered(walletID)).to.be
+          .true
+      })
+    })
+  })
+
+  describe("getWalletPublicKey", async () => {
+    context("with wallet not registered", async () => {
+      it("should revert", async () => {
+        await expect(
+          walletRegistry.getWalletPublicKey(formatBytes32String("NON EXISTING"))
+        ).to.be.revertedWith("Wallet with given ID has not been registered")
+      })
+    })
+
+    context("with wallet registered", async () => {
+      const walletPublicKey = ecdsaData.group1.publicKey
+
+      // TODO: Add tests to cover zero padding
```

Comment:
In this PR or separately?

Reply:
bfb8b9b7ad89e3dd0aa61af4837f7f545c75be98